### ISSUE TITLE
`waitForServices` times out after the first interval is reached since 0.14.22

### ISSUE THREAD

**Issue by rochdev** [moleculerjs/moleculer#1123]
## Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository

## Current Behavior

`waitForServices` times out after the first interval is reached.

## Expected Behavior

`waitForServices` should retry indefinitely unless a timeout is explicitly configured, which was the behavior before 0.14.22 and is also the [documented behavior](https://moleculer.services/docs/0.12/broker.html#Wait-for-services).

### Steps to Reproduce

1. Create two `ServiceBroker` instances.
2. Start both brokers.
3. Call `waitForServices` on one of the brokers with the name of the other broker, with the default timeout and interval.
4. Wait 1 second.

### Reproduce code snippet

```js
const port = 51817
const broker = new ServiceBroker({
  namespace: 'multi',
  nodeID: `server-${process.pid}`,
  logger: false,
  transporter: `tcp://127.0.0.1:${port}/server-${process.pid}`
})

broker.createService({
  name: 'math',
  actions: {
    add (ctx) {
      const numerify = this.actions.numerify

      return numerify(ctx.params.a) + numerify(ctx.params.b)
    },

    numerify (ctx) {
      return Number(ctx.params)
    }
  }
})

const clientBroker = new ServiceBroker({
  namespace: 'multi',
  nodeID: `client-${process.pid}`,
  logger: false,
  transporter: {
    type: 'TCP',
    options: {
      udpDiscovery: false,
      urls: [
        `127.0.0.1:${port}/server-${process.pid}`
      ]
    }
  }
})

broker.start()
clientBroker.start()

clientBroker.waitForServices('math')
```

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Moleculer version: 0.14.12
* NodeJS version: 16.15.1
* Operating System: macOS 12.5

### Failure Logs
```
MoleculerServerError: Services waiting is timed out.
```

**Comment by icebob** [comment]
Thanks the report, it was a wrong variable issue, but fixed.

**Comment by icebob** [comment]
Released as 0.14.23
